utils.py

```diff
@@ -19,6 +19,7 @@ def get_list(item, index):
 
 def get_id_list(param):
     student_list = list(param[get_keyword_value('student_list')])
+    exclude_list = list(param[get_keyword_value('exclude_items')])
```

Comment:
If the exclude_items object isn't in the json then it will give a key error
![image](https://user-images.githubusercontent.com/61991709/79054497-e1523600-7c0a-11ea-83a8-381e351ffb1c.png)